cmd/apichecker/apicheck.go

```diff
@@ -0,0 +1,1226 @@
+	pathHitBTC         = "https://api.hitbtc.com/"
+	pathBitflyer       = "https://lightning.bitflyer.com/docs?lang=en"
+	pathLakeBTC        = "https://www.lakebtc.com/s/api_v2"
+	pathKraken         = "https://www.kraken.com/features/api"
+	pathAlphaPoint     = "https://alphapoint.github.io/slate/#introduction"
+	pathYobit          = "https://www.yobit.net/en/api/"
+	pathLocalBitcoins  = "https://localbitcoins.com/api-docs/"
+	pathGetAllLists    = "https://api.trello.com/1/boards/%s/lists?cards=none&card_fields=all&filter=open&fields=all&key=%s&token=%s"
+	pathNewCard        = "https://api.trello.com/1/cards?%s&key=%s&token=%s"
+	pathChecklists     = "https://api.trello.com/1/checklists/%s/checkItems?%s&key=%s&token=%s"
+	pathChecklistItems = "https://api.trello.com/1/checklists/%s?fields=name&cards=all&card_fields=name&key=%s&token=%s"
+	pathUpdateItems    = "https://api.trello.com/1/cards/%s/checkItem/%s?%s&key=%s&token=%s"
+	complete           = "complete"
+	incomplete         = "incomplete"
+)
+
+// Config is a format for storing update data
+type Config struct {
+	CardID      string         `json:"CardID"`
+	ChecklistID string         `json:"ChecklistID"`
+	Key         string         `json:"Key"`
+	Token       string         `json:"Token"`
+	Exchanges   []ExchangeInfo `json:"Exchanges"`
+}
+
+var (
+	verbose                                                                        bool
+	apiKey, apiToken, trelloBoardID, trelloListID, trelloChecklistID, trelloCardID string
+	configData, testConfigData                                                     Config
+)
+
+func main() {
+	flag.StringVar(&apiKey, "key", "", "sets API key for Trello board interaction")
+	flag.StringVar(&apiToken, "token", "", "sets API token for Trello board interaction")
+	flag.StringVar(&trelloChecklistID, "checklistid", "", "sets checklist ID for Trello board interaction")
+	flag.StringVar(&trelloCardID, "cardid", "", "sets card ID for Trello board interaction")
+	flag.StringVar(&trelloListID, "listid", "", "sets list ID for Trello board interaction")
+	flag.StringVar(&trelloBoardID, "boardid", "", "sets board ID for Trello board interaction")
+	flag.BoolVar(&verbose, "verbose", false, "increases logging verbosity for API Update Checker")
+	flag.Parse()
+	var err error
+	testConfigData, err = ReadFileData(testJSONFile)
+	if err != nil {
+		log.Fatal(err)
+	}
+	configData, err = ReadFileData(jsonFile)
+	if err != nil {
+		log.Fatal(err)
+	}
+	if CanUpdateTrello() {
+		UpdateFile(&configData, backupFile)
+		if err != nil {
+			log.Fatal(err)
+		}
+		configData.Key = apiKey
+		configData.Token = apiToken
+		configData.CardID = trelloCardID
+		configData.ChecklistID = trelloChecklistID
+		err = CheckUpdates(jsonFile, &configData)
+		if err != nil {
+			log.Fatal(err)
+		}
+	} else {
+		log.Printf("This is a test update since API keys are not set.\n")
+		err := CheckUpdates(testJSONFile, &testConfigData)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
```

Comment:
at this point we might also just want to let the user know everything completed fine

if you don't have verbose on and run it just exits without any other output